sys/net/gnrc/Makefile.dep

```diff
@@ -46,12 +46,12 @@ ifneq (,$(filter gnrc_lorawan_1_1,$(USEMODULE)))
   FEATURES_REQUIRED += periph_flashpage
 endif
 
-ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
+ifneq (,$(filter gnrc_netif_dev_default,$(USEMODULE)))
```

Comment:
This should remain `gnrc_netif_dev_default`

Reply:
uh, that's a result of `sed`. I will revert it.